Software/src/qtsingleapplication/common.pri

```diff
@@ -1,4 +1,4 @@
-exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib
+#exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib

```

Comment:
I can't solve it. For some reason qmake ignores `exist` for `infile`. I can either leave it this way, or add an empty `config.pri`. What do you prefer?